src/gluonts/model/r_forecast/_predictor.py

```diff
@@ -207,5 +207,9 @@ def predict(
                 else None
             )
             yield SampleForecast(
-                samples, forecast_start(data), self.freq, info=info
+                samples,
+                forecast_start(data),
+                self.freq,
+                info=info,
+                item_id=entry.get("item_id", None),
```

Comment:
```suggestion
                item_id=entry.get("item_id"),
```

Should also work, shouldn't it?

Reply:
Ah yes true, I just did it in the same way as in #1113 and #1173